finvizfinance/quote.py

```diff
@@ -164,6 +164,16 @@ def ticker_fundament(self, raw=True, output_format="dict"):
                             fundament_info[header] = True
                         else:
                             fundament_info[header] = False
+                    # Adding this code for the EPS Next Y, because there is same two keys, one is percentage and other one is Dollar Ammount. This code will get both.
```

Comment:
I think this should determine by the last char of the value (%) not by whether it already exists.

Reply:
Yes, you are right. But what if the value is empty and the output is "-". So, it will not include the EPS This Y in the dictionary again and it will raise the KeyError. So, I thought the best and possible way to solve this problem.

Reply:
Like if the value is - and some one access the EPS This Y Percentage. It will raise the key error.

Reply:
I see. Then just change the comment. Maybe like handle EPS Next Y key.

Reply:
Yes, I have make it shorter, like I have handled in the else condition. So the code is short, let me again pull the request with good comment.